### DISCUSSION TITLE
URL Links in Neovim

### DISCUSSION THREAD

**Discussion by andyl** [ghostty-org/ghostty#4203] [Q&A] [answered]
Using "ctrl-click" to open URLs is a killer feature.  On my Linux box, it works for local files, and also when I'm SSHed to a remote machine.  

However,  when I'm editing a file with Neovim, "ctrl-click" does not work.  If there is a url "https://mysite.com", and my cursor hovers over 'https', "ctrl-click" raises a Neovim error "Tag not found: https".

Anyone know how to configure Neovim and/or Ghostty to make this work?

**Comment by jcollie** [comment, marked as answer]
When you're in an editor (or anything else that uses the alternate buffer) use `ctrl+shift+click`.
